commit 1f7f035528bbbacbe53b3fb8b1fdd71130e84f0d (refs/changes/03/4403/1)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2018-06-21T19:28:10+00:00 (1 year, 4 months ago)
    
    TRITON-535 Need to update sdc-sdc's sdc-napi-ufds-watcher submodule

diff --git a/deps/sdc-napi-ufds-watcher b/deps/sdc-napi-ufds-watcher
index 1bc4077..2d4e757 160000
--- a/deps/sdc-napi-ufds-watcher
+++ b/deps/sdc-napi-ufds-watcher
@@ -1 +1 @@
-Subproject commit 1bc40777f03ae3a2d6e5eabc2b441c9c7cd8404d
+Subproject commit 2d4e75711d2212d808bb167b0b3c8d9eecb7d7f1
